e2e/ansible/playbooks/utils/copy_task.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: Copy the chaoskube specs to kubemaster
```

Comment:
I guess you meant it to be just "specs" or 'files"

Reply:
Yeah, will correct that.